packages/app-content-pages/package.json

```diff
@@ -20,7 +20,7 @@
     "@zooniverse/async-states": "~0.0.1",
     "@zooniverse/grommet-theme": "~3.2.0",
     "@zooniverse/panoptes-js": "~0.5.0",
-    "@zooniverse/react-components": "~1.10.0",
+    "@zooniverse/react-components": "~1.10.1",
```

Comment:
This change isn't really necessary. `~1.10.0` will install the latest version of 1.10, because of the tilde.
https://docs.npmjs.com/cli/v6/configuring-npm/package-json#dependencies